lib/db.js

```diff
@@ -278,9 +264,9 @@ DB.prototype.blockHandler = function(block, add, callback) {
   }
 
   async.eachSeries(
-    this.modules,
-    function(module, next) {
-      module['blockHandler'].call(module, block, add, function(err, ops) {
+    this.node.modules,
```

Comment:
Yeah, primarily because it's better for accessing a module by name (as needed in `start()`):
```js
self.modules[service.name].start(next);
```